### DISCUSSION TITLE
Material Colorization

### DISCUSSION THREAD

**Discussion by Aetherinox** [Taiizor/ReaLTaiizor#114] [Q&A] [answered]
I've read a few posts about this subject, but it's not completely clear.
You have color options listed within the source code, but it is for particular colors.

Is their any way to modify an XML or some class which allows me to customize the colors to my own specific needs using either HEX or RGB. Or am I restricted to only the class of colors you've specified.

I saw something related to the metro theme, but I cannot find this for the Material theme.

**Comment by Taiizor** [comment, marked as answer]
It's there for the Metro theme yes, but the same is not true for the Material theme. You only have to use the predefined colors.

**Comment by Aetherinox** [comment]
Thanks.

Not sure what the future plans are, but it would really be a huge help if this could be a possible feature in the future. I like some of the controls, but our team has very specific color schemes we use. And while material is nice, I'm more interested in customization. 

Also noticed for TabControl, the blue at the top does not change. I have TabControl's ForeColor set to a neon yellow to test with, and the tab when running the program is still blue. 

Also, Text Unselected and Selected are set to their defaults, which is dark grey and White respectively, but they stick to dark grey as the only color (for TabControl).

Not sure if you want these added as a bug, or if there's some secret magic to this, but I have the metro style theme added properly with the metroStyleManager.

![yJuSCfr](https://github.com/Taiizor/ReaLTaiizor/assets/118329232/2c28a9be-035e-44b6-b8be-f3c9c106dd74)

**Comment by Taiizor** [comment]
> Also, Text Unselected and Selected are set to their defaults, which is dark grey and White respectively, but they stick to dark grey as the only color (for TabControl).

SelectedTextColor & UnselectedTextColor should now work.
![image](https://github.com/Taiizor/ReaLTaiizor/assets/41683699/ff94f557-3614-48ac-9e6e-3b9366f79b5c)
![image](https://github.com/Taiizor/ReaLTaiizor/assets/41683699/fd94b1ac-202a-42e7-9998-a80eb8d9e478)

**Comment by Taiizor** [comment]
> Also noticed for TabControl, the blue at the top does not change. I have TabControl's ForeColor set to a neon yellow to test with, and the tab when running the program is still blue.

Because only when Style Custom is selected, the customized colors are valid.

![image](https://github.com/Taiizor/ReaLTaiizor/assets/41683699/7a412d66-9cb4-4b69-9493-4ec46c5779c2)
![image](https://github.com/Taiizor/ReaLTaiizor/assets/41683699/37ca7a89-ade6-49a6-9ecd-9eb451f9888c)
